library/ic-ledger-types/src/lib.rs

```diff
@@ -203,11 +217,17 @@ pub struct Memo(pub u64);
 /// Arguments for the `transfer` call.
 #[derive(CandidType, Serialize, Deserialize, Clone, Debug)]
 pub struct TransferArgs {
+    /// The memo for the transaction.
     pub memo: Memo,
+    /// The amount to be transferred.
     pub amount: Tokens,
+    /// The expected fee when transferring. Should be 0.0001 ICP.
```

Comment:
Suggestion: crib from https://github.com/dfinity/ic/blob/master/rs/rosetta-api/icp_ledger/ledger.did#L43 (this and other fields)

```suggestion
    /// The amount that the caller pays for the transaction.
    /// Must be 10000 e8s.
```